scripts/travis_script.sh

```diff
@@ -38,6 +38,10 @@ if [[ $GROUP == coverage_and_docs ]]; then
     # Run the coverage check.
     npm run test:coverage
 
+    # Run the link check
+    pip install -q pytest-check-links
+    py.test --check-links **/*.md
```

Comment:
Oh, sorry. It should be `py.test --check-link *.md **/*.md`, as `**` seems to map 1-or-more subdir on bash, not 0-or-more.